.github/workflows/reusable-build-test-release.yml

```diff
@@ -14,6 +14,11 @@ on:
         type: string
         default: >-
           [""]
+      custom-version:
+        required: false
+        description: 'Version of release specified by user instead of semantic release'
+        type: string
```

Comment:
Can any string be passed in `custom-version` input?

Reply:
yes

Reply:
Should we think of some form of validation then?

Reply:
Validation added in `validate-custom-version` job

Reply:
It would be best to check if the version you are submitting already exists, but I don't think it blocks this feature.